src/components/app-top-bar.tsx

```diff
@@ -72,32 +71,24 @@ export default function AppTopBar({ userManagerInstance }: AppTopBarProps) {
     }, [user]);
 
     return (
-        <>
-            <TopBar
-                appName={APP_NAME}
-                appColor="#3DABE2"
-                appLogo={theme === LIGHT_THEME ? <GridExploreLogoLight /> : <GridExploreLogoDark />}
-                appVersion={AppPackage.version}
-                appLicense={AppPackage.license}
-                onLogoutClick={() => logout(dispatch, userManagerInstance)}
-                onLogoClick={() => navigate('/', { replace: true })}
-                user={user ?? undefined}
-                appsAndUrls={appsAndUrls}
-                onThemeClick={handleChangeTheme}
-                theme={themeLocal}
-                onLanguageClick={handleChangeLanguage}
-                language={languageLocal}
-                globalVersionPromise={() => fetchVersion().then((res) => res?.deployVersion)}
-                additionalModulesPromise={getServersInfos as () => Promise<GridSuiteModule[]>}
-            >
-                {user && <SearchBar inputRef={searchInputRef} />}
-            </TopBar>
-            <ParametersDialog showParameters={showParameters} hideParameters={() => setShowParameters(false)} />
```

Comment:
why did you remove ParametersDialog ?

Reply:
it's not used, showParameters is always false